### PR TITLE
Doc update with interval meaning and shard example for sockets.

### PR DESCRIPTION
**Pull request by Hiryus**
See diff.

### CODE REVIEW

File: docs/Websocket_endpoints.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ visual   | Object | Room visuals (contents currently unknown #FIXME)
 
 ```javascript
 // Subscription
-api.socket.subscribe('room:W7N7', event => console.log(JSON.stringify(event.data)));
+api.socket.subscribe('room:shard0/W7N7', event => console.log(JSON.stringify(event.data)));
```
Comment (ags131) [inline docs/Websocket_endpoints.md]: A note here, on non sharded servers (IE all current private servers), its still the old way.

Comment (Hiryus) [inline docs/Websocket_endpoints.md]: Not sure if `Note that adding a shard to a server not expecting it may cause an error.` is entirely true...

**Comment by Hiryus** [comment]
Not sure if `Note that adding a shard to a server not expecting it may cause an error.` is entirely true...

**Comment by ags131** [comment]
Technically, the sevrer will just silently ignore it, but meh

**Pull request merged**